DESCRIPTION

```diff
@@ -31,8 +31,11 @@ Imports:
     digest,
     crosstalk,
     reactR,
-    purrr
+    purrr,
+    withr (>= 2.4.2)
```

Comment:
Does this need to be in `Imports` if it's only being used for tests?

Reply:
Good point.  I just used usethis::use_dev_package() and this is where it put it.  Curious.